pkg/workloads/cortex/serve/start/batch.py

```diff
@@ -41,9 +41,6 @@
     "job_spec": None,
     "provider": None,
     "predictor_impl": None,
-    "predict_route": None,
-    "client": None,
-    "class_set": set(),
```

Comment:
i noticed you are removing these fields on a BatchAPI file, is this intentional?